ui/app/models/node.js

```diff
@@ -47,8 +47,10 @@ export default Model.extend({
   completeAllocations: computed('allocations.@each.clientStatus', function() {
     return this.allocations.filterBy('clientStatus', 'complete');
   }),
-  runningAllocations: computed('allocations.@each.isRunning', function() {
-    return this.allocations.filterBy('isRunning');
+  runningAllocations: computed('allocations.@each.clientStatus', function() {
+    return this.allocations.filter(
+      a => a.get('clientStatus') === 'pending' || a.get('clientStatus') === 'running'
+    );
```

Comment:
If the definition of running for an allocation should include pending (which is reasonable) I think it should be updated in the allocation model instead of here.

https://github.com/hashicorp/nomad/blob/16d076d542155a43fee478e9a86e105b21ffac4a/ui/app/models/allocation.js#L43

Reply:
I'm afraid of making that change in the model, for fear of unintended side-effects.  Other callers to isRunning, e.g. Alloc Stats reporting, seem only relevant when the alloc is actually running and not pending.  So this seems like the smallest scope for change.